### PR TITLE
Default testalert to never

### PR DESCRIPTION
**Pull request by maxbartunik**
Currently all alerts set to default values are taken as testalerts on a Sunday. This PR fixes the issue and defaults to no testalert.

### CODE REVIEW

File: src/model/zvei.mjs [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default class ZVEI {
         }
         this.description = description;
 
-        if (test_day < 0 || test_day >= 7) {
+        if (test_day < -1 || test_day >= 7) {
             throw new Error(`Test day '${test_day}' is not from the range [0,6]`);
```
Comment (keszocze) [inline src/model/zvei.mjs:28]: As you allow `-1` to be a valid date, the error message should reflect this.
```suggestion
            throw new Error(`Test day '${test_day}' is not from the range [-1,6]`);
```

Btw, are you sure that `-1` is not the second to last day of the week? I mean, JS does weird things with dates.

**Comment by keszocze** [comment]
From neither the description nor the code changes I have any idea, what this request is about.

**Comment by maxbartunik** [comment]
> From neither the description nor the code changes I have any idea, what this request is about.

Sorry, thiw was a middle of the night thing. We currently have a bug: As "Schleifen" default to testday 0, testtime 0 to 0, all ZVEIs, that do not have a testperiod set are detected as "test alerts" all day on sundays. This is counter intuitive (i.e. wrong). Instead alert with no testperiod set should never be detected as "test alert". This is fixed by this PR.

**Comment by maxbartunik** [comment]
The changes are:
- change default test day to -1 (i.e. never)
- allow -1 as parameter in test cases etc.

**Comment by keszocze** [comment]
> The changes are:
> 
>     * change default test day to -1 (i.e. never)
> 
>     * allow -1 as parameter in test cases etc.

I really hate "magic numbers" such as `-1` meaning something. I always prefer something like an optional in such a case. But I think that is I fight I already lost $\Rightarrow$ accepting

**Comment by maxbartunik** [comment]
> > The changes are:
> > ```
> > * change default test day to -1 (i.e. never)
> > 
> > * allow -1 as parameter in test cases etc.
> > ```
> 
> I really hate "magic numbers" such as `-1` meaning something. I always prefer something like an optional in such a case. But I think that is I fight I already lost ⇒ accepting
> 
> (you do have to change the error message though...)

true - I am trying but frequently fall back ;)

**Review by keszocze — CHANGES_REQUESTED** [review]

**Comment by maxbartunik** [comment]
> > > The changes are:
> > > ```
> > > * change default test day to -1 (i.e. never)
> > > 
> > > * allow -1 as parameter in test cases etc.
> > > ```
> > 
> > 
> > I really hate "magic numbers" such as `-1` meaning something. I always prefer something like an optional in such a case. But I think that is I fight I already lost ⇒ accepting
> > (you do have to change the error message though...)
> 
> true - I am trying but frequently fall back ;)

I will leave this for v2...

**Pull request merged**